cmd/glasskube/cmd/version.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/glasskube/glasskube/internal/cliutils"
+	"github.com/glasskube/glasskube/internal/config"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var versioncmd = &cobra.Command{
+	Use:    "version",
+	Short:  "Print the version of glasskube and package-operator",
+	Long:   `Print the version of glasskube and package-operator`,
+	PreRun: cliutils.SetupClientContext(true),
```

Comment:
The version cmd should also work in a non-bootstrapped cluster, but this is prevented here. To change this, you can change this to `cliutils.SetupClientContext(false)`